packages/fxa-shared/l10n/determineLocale.ts

```diff
@@ -0,0 +1,15 @@
+import { parseAcceptLanguage } from './parseAcceptLanguage';
```

Comment:
This could use a license at the top as well as `parseAcceptLanguage.ts`.

Reply:
Good catch. Thanks.